src/components/ButtonTrust.js

```diff
@@ -0,0 +1,65 @@
+import { ButtonGroup } from '@material-ui/core';
+import { makeStyles } from '@material-ui/core/styles';
+import PropTypes from 'prop-types';
+import React from 'react';
+
+import { SEARCH_PATH, SEND_PATH } from '~/routes';
+
+import Button from '~/components/Button';
+// import translate from '~/services/locale';
+
+const useStyles = makeStyles((theme) => ({
+  buttonContainer: {
+    marginTop: '30px',
+    marginBottom: '70px',
+    padding: '0 16px',
+    '& a:first-of-type': {
+      border: 0,
+      marginRight: '4px',
+      padding: '6px 15px',
+      '&.Mui-disabled': {
+        borderTopRightRadius: 0,
+        borderBottomRightRadius: 0,
+      },
+    },
+    '& a:nth-of-type(2)': {
+      borderLeftStyle: 'none',
+      marginLeft: '4px',
+      padding: '6px 15px',
+      border: 0,
+      '&.Mui-disabled': {
+        borderTopLeftRadius: 0,
+        borderBottomLeftRadius: 0,
+      },
+    },
+  },
+
+  buttonGradientOpposite: {
+    background: theme.custom.gradients.purpleOpposite,
+    '&:hover': {
+      background: theme.custom.gradients.purpleOppositeHover,
+    },
+  },
+}));
+
+const ButtonTrust = ({ leftBtnText, rightBtnText }) => {
```

Comment:
This name is not accurate. It is not used for TurstButtons only and even if it was atm it wouldn't be future proof. I suggest ButtonDouble or similar, describing the functionality and not one o its usecases

Reply:
corrected